main.go

```diff
@@ -54,6 +55,26 @@ func main() {
 		case 4:
 			log.Infof("- %s cancelled %s", build.TriggeredWorkflow, buildURL)
 		}
+		if build.Status != 0 {
+			if cfg.SavePath != "" {
```

Comment:
please use a strings.TrimSpace on paths, users might leave a space at the and of the line or so